### ISSUE TITLE
Beta dot animation breaking fonts subpixel antialiasing on Windows in Chrome

### ISSUE THREAD

**Issue by Palid** [vector-im/element-web#18564]
It seems that the animation `mx_Beta_bluePulse` used in class `mx_BetaDot` is making an enormous layout trashing and is a second source of fonts not being correctly subpixel antialiased on Windows. The main source so far is the background-blur on left panel, which is being taken care of here: https://github.com/matrix-org/matrix-react-sdk/pull/6262

To reproduce proper behavior comment out class `.mx_BetaDot` in `res/css/views/beta/_BetaCard.scss` and remove all `backdrop-filter` references from our css codebase. 
The other way to verify if rendering is proper is to look for layer borders checkbox which is accessible via Chrome Devtools cog menu (Rendering). Screenshots below:

Valid fonts rendering (properly antialiased) will have this kind of layer borders: 
![CleanShot 2021-08-16 at 11 58 40](https://user-images.githubusercontent.com/3636685/129546467-c1f2c807-173b-4555-a61e-d8e96023c59a.png)

Invalid fonts rendering layer borders (it's super easy to just look at the bottom-right corner with icons):
![CleanShot 2021-08-16 at 11 59 40](https://user-images.githubusercontent.com/3636685/129546494-70f4fcfe-752b-492a-8e69-56ab3f82f979.png)

The fonts rendering is irreproducible on anywhere except Windows and Chrome/Chromium, yet the layers part should be easily reproducible on any system using Chrome.

Related: https://github.com/matrix-org/matrix-react-sdk/pull/6262 (gets rid of background-blur and replaces it with canvas, which doesn't trash the layout)
Related: https://github.com/vector-im/element-web/issues/15594

**Comment by Palid** [comment]
Revisit in a week, this is either already solved, or will be automatically closed after we release the version without beta dot.

**Comment by Palid** [comment]
Seems to be sorted out by the DOM rework.
